test/integration.js

```diff
@@ -39,7 +39,11 @@ fs.readdir(integration, (err, files) => {
                         is_json = false;
                         //process.exit(1);
                     }
-                    fs.writeFileSync(path.join(integration, 'actual', file + ".log"), stdout);
+                    var actualDir = path.join(integration, 'actual');
```

Comment:
This if could be moved to the start - there is no reason to check this folder for every test.